src/languages/postgresql.formatter.ts

```diff
@@ -1659,9 +1668,25 @@ const binaryOperators = [
   '@>',
   '<@',
   '~=',
+  // Network Address operators
   '>>=',
   '<<=',
+  // Text Search Operators
   '@@@',
+  // JSON Operators
+  '@?',
+  '?&',
+  '->',
+  '->>',
+  '#>',
+  '#>>',
+  '#-',
+  // Other Operators
+  ':=',
+  // Unknown
+  '::',
+  '=>',
+  '=#',
```

Comment:
So, what's up with these, and what's the difference between "Other" and "Unknown"?

- `:=` is [assignment](https://www.postgresql.org/docs/current/plpgsql-statements.html#PLPGSQL-STATEMENTS-ASSIGNMENT) or for named arguments
- `::` is a type cast
- `=>` is for named function arguments #126  
- `=#` well, you just added it, so you better know what it's for :)